### ISSUE TITLE
indexed_colors: missing field `color`

### ISSUE THREAD

**Issue by brvn0** [catppuccin/alacritty#4]
```
[0.000644911s] [ERROR] [alacritty_config_derive] Config error: indexed_colors: missing field `color`
[130.229424901s] [ERROR] [alacritty_config_derive] Config error: indexed_colors: missing field `color`
```
It's an error with the latest commit; I tried using the config from before the last commit and it works there
I'm sorry for just submitting an issue and not a pull request but I have absolutely no idea about the alacritty styling xd 

Reproduce:
`curl https://raw.githubusercontent.com/catppuccin/alacritty/main/catppuccin.yml > .config/alacritty/alacritty.yml`

Temporary fix:
`curl https://raw.githubusercontent.com/catppuccin/alacritty/2b7b781ce27398a9eb4695ee250eb096fdfbfab4/catppuccin.yml > .config/alacritty/alacritty.yml`

Edit: Added the reproduce and temp. fix commands

**Comment by mimame** [comment]
Hi @brvn0,
I did a pull request with the fix here https://github.com/catppuccin/alacritty/pull/5
I'm sure @andreasgrafen will take care of this very soon
Have a nice day!

**Comment by andreasgrafen** [comment]
@brvn0 @mimame
I think this has been fixed with Commit [d337346](https://github.com/catppuccin/alacritty/commit/d337346afae8d2f3c99d84c874b4e356e443ca0f), no? But please feel free to correct me if I'm wrong … c:

**Comment by brvn0** [comment]
Yup works now :D
